.travis.yml

```diff
@@ -7,7 +7,7 @@ language: java
 jdk:
   - openjdk8
 
-script: "mvn cobertura:cobertura"
+script: "./gradlew jacocoTestReport"
```

Comment:
we also need to add the command to run the tests. the jacocoTestReport only runs the code coverage tool. We also need to run `./gradlew test`